### PR TITLE
Update 3 example descriptions to newer format

### PR DESCRIPTION
**Pull request by EmmaJP**
Minor updates to 3 of the example descriptions to the newer format requirements, in https://act-rules.github.io/rules/6a7281 - ARIA state or property has valid value. 

Need for Final Call:
- no need for final call.

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Comment by EmmaJP** [comment]
@WilcoFiers Mira and I didn't go further because I didn't know if Passed Example 4 needed an element with the referenced id in the associated code snippet. Potentially a change that might need a final call, so not sure how you want to proceed.

Also, I don't know Mira's github id to assign the PR to her.

**Review by ajanec01 — DISMISSED** [review]

File: _rules/aria-state-or-property-valid-value-6a7281.md [hunk 1]
```diff
@@ -60,23 +60,23 @@ Some user agents treat the value of `aria-*` attribute as case-sensitive (even w
...
+This `aria-expanded` attribute has a valid true/false/undefined value.
 
 ```html
 <div role="button" aria-expanded="undefined">A button</div>
 ```
 
 #### Passed Example 3
 
-`aria-pressed` state with valid tristate value
+This `aria-pressed` attribute has a valid true/false/mixed/undefined value.
```
Comment (Jym77) [inline _rules/aria-state-or-property-valid-value-6a7281.md:79]:
```suggestion
This `aria-pressed` attribute has a valid tristate value.
```
"tristate" is the correct ARIA term for this attribute: https://www.w3.org/TR/wai-aria-1.2/#aria-pressed

**Review by Jym77 — DISMISSED** [review]
I'm not sure about the change from "property/state" to "attribute". This difference does exist in ARIA, so the terms are a bit more precise than just "attribute". Not a blocker, however.

File: _rules/aria-state-or-property-valid-value-6a7281.md [hunk 2]
```diff
@@ -66,39 +66,39 @@ For value type `URI`, this rule does not require that the destination URI exists
 
 #### Passed Example 1
 
-This `aria-required` attribute has a valid true/false value.
+This `aria-required` property has a valid true/false value.
```
Comment (Jym77) [inline _rules/aria-state-or-property-valid-value-6a7281.md:69]:
```suggestion
This `aria-required` property has a valid [true/false](https://www.w3.org/TR/wai-aria-1.2/#valuetype_true-false) value.
```

**Review by Jym77 — APPROVED** [review]
We may want to link the value types to [their definitions](https://www.w3.org/TR/wai-aria-1.2/#propcharacteristic_value).

Not sure about it though (the same way we don't link element's or attribute's name to their HTML def).

**Review by WilcoFiers — APPROVED** [review]

**Review by colabottles — APPROVED** [review]

**Comment by WilcoFiers** [comment]
This PR is completely out of date. All tests have since been rewritten.

**Pull request closed without merging**